common/PacketTypes.asn1

```diff
@@ -0,0 +1,29 @@
+PacketTypes DEFINITIONS ::= BEGIN
+EXPORTS ALL;
+IMPORTS
+    PUSC-UINT, PUSC-BOOL FROM BasicTypes
+    ApplicationProcess-ID FROM ApplicationProcess;
+
+
+ApplicationProcess-ID-Packet ::= ApplicationProcess-ID
+SecondaryHeaderFlag ::= PUSC-BOOL
+
+Packet-ID {PacketType} ::= SEQUENCE
+{
+    packetType PacketType,
+    secondaryHeaderFlag SecondaryHeaderFlag,
+    applicationProcess-ID ApplicationProcess-ID-Packet
+}
+
+TM-Packet-ID ::= Packet-ID {NULL}
+TC-Packet-ID ::= Packet-ID {NULL}
+
+PacketSequenceControl ::= SEQUENCE
+{
+    sequenceFlags NULL,
```

Comment:
this should not be a `NULL` (it might be in PROBA3 but in standard it isn't :) )

Reply:
Seems ok to me.

"The sequence flags are defined by CCSDS but not used by the space packet protocol.
This Standard uses the binary value "11" for the sequence flags, to indicate a standalone
packet. All telemetry packets and telecommand packets defined within
this Standard are stand-alone packets."

Reply:
ohh... I missed that in new standard, nice (old one treated those flags similar, but did not mention it explicitly)